src/Uno.UI/UI/Xaml/FrameworkElement.Layout.crossruntime.cs

```diff
@@ -168,6 +168,11 @@ void MeasureCoreWithTrace(Size availableSize)
 
 		private void InnerMeasureCore(Size availableSize)
 		{
+			if (_traceLayoutCycle && this.Log().IsEnabled(LogLevel.Error))
+			{
+				this.Log().LogError($"[LayoutCycleTracing] Measuring {this},{this.GetDebugName()} with availableSize {availableSize}.");
```

Comment:
Enable structured logging...  (+others in this PR)

```suggestion
				this.Log().LogError("[LayoutCycleTracing] Measuring {Source},{Name} with availableSize {AvailableSize}.", this, this.GetDebugName(), availableSize);
```

Reply:
Named placeholders are not gonna work. This method just does a simple `string.Format`.

We can use `{0}`, `{1}`, `{2}`, but I think interpolation is easier to read.

Reply:
Yes; interpolqtion is easier to read.